canonical-web-and-design/charmhub.io
# New bundle cards

## Done
- Initial bundle cards
- When the browser resizes the number of charms, adjusts for the space – this is not 100% accurate but is more cautious
- *NOTE FOR UX AND DESIGN* The design in https://app.zeplin.io/project/623c85b9b27d9fa62165f655/screen/624d4b3079f687143e35527e shows 3 rows, but doesn't consider bundles that have double-line names. As such, I've limited the maximum charm rows to 2. Happy to adjust if need be.

## How to QA
- Visit https://charmhub-io-1378.demos.haus/ and use the main listing page to see some bundles. Filter, search, etc.
- A good candidate to search for is "kubeflow"
- "openstack" also has bundles with lot's of icons

## Issue / Card
Fixes #1359 

## Screenshots
![image](https://user-images.githubusercontent.com/479384/180978637-83d7f92c-ca00-4bf5-8174-6fb8882a04ea.png)
